File: src/main/java/org/jukeboxmc/event/player/PlayerJoinEvent.java
package org.jukeboxmc.event.player;

import org.jukeboxmc.player.Player;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class PlayerJoinEvent extends PlayerEvent {

    private String joinMessage;

    public PlayerJoinEvent(Player player, String joinMessage) {
        super(player);

        this.joinMessage = joinMessage;
    }

    public String getJoinMessage() {
        return this.joinMessage;
    }

    public void setJoinMessage(String joinMessage) {
        this.joinMessage = joinMessage;
    }
}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/event/player/PlayerJoinEvent.kt
package org.jukeboxmc.api.event.player

import org.jukeboxmc.api.player.Player

class PlayerJoinEvent(
    player: Player,
    private var joinMessage: String? = null
) : PlayerEvent(player){

    fun getJoinMessage(): String? {
        return this.joinMessage
    }

    fun setJoinMessage(joinMessage: String?) {
        this.joinMessage = joinMessage
    }

}